packages/manager/modules/netapp/src/routing.js

```diff
@@ -42,7 +42,7 @@ export default /* @ngInject */ ($stateProvider) => {
       apiPath: () => '/storage/netapp',
       dataModel: () => 'storage.NetAppService',
       defaultFilterColumn: () => 'id',
-      header: () => 'NetApp',
+      header: () => 'Enterprise File Storage',
```

Comment:
The name sounds English compared to "NetApp" which is just a product name. Are you sure this is not a key that must be translated ?

Reply:
No, it's already translated like this in others files.

Reply:
you should use angular translate here, see : https://github.com/ovh/manager/blob/master/packages/manager/modules/emailpro/src/email-pro.routing.js#L13

Reply:
IMHO we shouldn't use translations here because the Product name is not meant to be translated into other languages.